javascript-client/test/api/OrderApi.spec.js

```diff
@@ -49,27 +51,32 @@
       object[property] = value;
   }
 
-  describe('OrderApi', function() {
-    describe('orderAddPartIdPost', function() {
-      it('should call orderAddPartIdPost successfully', function(done) {
-        //uncomment below and update the code to test orderAddPartIdPost
-        //instance.orderAddPartIdPost(function(error) {
-        //  if (error) throw error;
-        //expect().to.be();
-        //});
-        done();
+  describe('OrderApi', function () {
+    describe('End to End', function () {
+      it('Non admin user should add order to cart and list all orders', async function (done) {
+        const userToken = await authUtil.loginUser(constants.nonAdminUser.email, constants.nonAdminUser.password);
```

Comment:
Now by looking at this code I think that you can have 2 loginUser methods. One that accepts username and password and another that accepts User object.
The one that takes User object as argument can call the other :)